components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { adminMenu } from "./admin-menu";
+
+import { LicenseContext } from "../license-context";
+import { ReactElement, useContext, useEffect } from "react";
+import { getGitpodService } from "../service/service";
+
+import { ReactComponent as Alert } from "../images/exclamation.svg";
+import { ReactComponent as Success } from "../images/check-circle.svg";
+import { LicenseInfo } from "@gitpod/gitpod-protocol";
+import { ReactComponent as XSvg } from "../images/x.svg";
+import { ReactComponent as CheckSvg } from "../images/check.svg";
+import SolidCard from "../components/SolidCard";
+import Card from "../components/Card";
+
+export default function License() {
+    const { license, setLicense } = useContext(LicenseContext);
+
+    useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
+        }
+        (async () => {
+            const data = await getGitpodService().server.adminGetLicense();
+            setLicense(data);
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
```

Comment:
NIT: Shouldn't it mean “no user limit”? :thinking: 
```suggestion
    // if user seats is 0, it means that there is no user limit in the license
```